compiler/ilgen/MethodBuilder.hpp

```diff
@@ -73,7 +74,7 @@ class MethodBuilder : public TR::IlBuilder
 
    TR::TypeDictionary *typeDictionary()                      { return _types; }
 
-   const char *getDefiningFile()                             { return _definingFile; }
+   std::string getDefiningFile()                             { return _definingFile; }
```

Comment:
should this return a reference or did you intend to make a copy? If you intended it, where will the copy be allocated?